cmd/tke-installer/app/installer/installer.go

```diff
@@ -576,6 +578,21 @@ func (t *TKE) initSteps() {
 			Func: t.postInstallHook,
 		},
 	}...)
+
+	t.steps = filterSteps(t.steps, t.Para.Config.SkipSteps)
+}
+
+func filterSteps(steps []handler, skips []string) []handler {
```

Comment:
Consider use https://github.com/thoas/go-funk#funkdifference ?

Reply:
funk.Difference does not work, use funk.Filter and funk.ContainsString instead.